Tests/travis/functions.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+getChangedFiles() {
+  git diff --name-only HEAD^
+}
+
+getAddedLines() {
+  file="$1"
+
+  git diff HEAD^ -- "$file" | grep -v "^+++ b/$file" | grep -e '^+' | sed -e 's/^+//'
+}
+
+getLinesFromFile() {
+  file="$1"
+  lines="$2"
+
+  echo "$lines" | sort -u | while IFS= read -r l; do
+      grep -n "^$l\$" -- "$file"
+  done
```

Comment:
`sort -u` is to be sure that every line will be found only once, right?
Won\`t it make `grep` output out of order? Maybe sort and make unique the result (line numbers), not lines themselves?

*(Didn\`t try it yet, so just a thought)*

Reply:
If we don't make line unique, then grep will do work twice.
I'll add ordering by line number into output.